src/llama_stack_client/_client.py

```diff
@@ -91,13 +78,19 @@ def __init__(
         # outlining your use-case to help us decide if it should be
         # part of our public interface in the future.
         _strict_response_validation: bool = False,
+        provider_data: Mapping[str, str] | None = None,
```

Comment:
I think it probably should be `Mapping[str, Any]` because some of the keys could have simple recursive list / dict like structure.